_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<footer class="darkest">
+<footer class="darkest dark:darkest">
```

Comment:
Why is this needed?

Reply:
Hm, good question. Looks like it is redundant. I played around with footer being dark:dark and later changed it.